packages/messages/src/browser/notifications.ts

```diff
@@ -71,6 +72,9 @@ export class Notifications {
         text.innerText = properties.text;
         const close = () => {
             element.remove();
+            if (properties.timeoutFn) {
```

Comment:
as the `close` function is called for each action, `timeoutFn` might do the same as the `onCloseFn`, right? please move that call to the `setTimeout` handler after L.85.